dtfs-central-api/src/services/repositories/utilisation-reports-repo.test.js

```diff
@@ -37,6 +96,34 @@ describe('utilisation-reports-repo', () => {
         id: '123',
         name: 'test user',
       },
+    }, {
+      bank: {
+        id: '124',
+        name: 'test bank',
+      },
+      month: 4,
+      year: 2021,
+      dateUploaded: expect.any(Date),
+      path: 'test path',
+      uploadedBy: {
+        id: '123',
+        name: 'test user',
+      },
+    }];
+
+    it('filters and sorts the data correctly', async () => {
```

Comment:
I don't think we should mention `filters` here as we are mocking the db collection response so the data is not filtered. I guess we can only do this in tests where an actual DB is spun up?

Reply:
Yep, very true